locale/ru/translations.json

```diff
@@ -453,7 +453,7 @@
   "Education (non-profit)": "Образование (некоммерческое)",
   "Effect": "Результат",
   "Either this user does not exist, or you do not have access to their profile.": "",
-  "Email": "Email",
+  "Email": "Электронная почта",
```

Comment:
Thanks for working on this! But translation changes need to be done in Transifex; see 
https://zulip.readthedocs.io/en/latest/translating/internationalization.html and the related guides.

Reply:
I can remove this change, no problem! I'll try to have it by tomorrow if it's no issue.

Reply:
@timabbott I am waiting for confirmation to change this. Please let me know if i should do this.

Reply:
Hey @IsabelaPO, please go ahead and remove any translation changes.